content/guides/deploying-your-app.mdx

```diff
@@ -0,0 +1,384 @@
+# Deployment Options
+
+Are you ready to deploy your Solid application? Follow our guides to different deployment services.
```

Comment:
```suggestion
Are you ready to deploy your Solid application? Follow our guides for different deployment services.
```

Reply:
While not wrong, "guides to" doesn't sound quite right to me. "Guides for" sounds and flows better, in my opinion. Feel free to disregard if you wish.